test/functional/services/inspector.ts

```diff
@@ -195,11 +195,20 @@ export class InspectorService extends FtrService {
    */
   public async openInspectorView(viewId: string): Promise<void> {
     this.log.debug(`Open Inspector view ${viewId}`);
+    this.log.debug(`view = ${viewId.substring(20)}`);
```

Comment:
Do we want to keep this debug statement?

Reply:
Experimental code. Thanks!